azure/services/roleassignments/roleassignments.go

```diff
@@ -32,12 +31,14 @@ import (
 )
```

Comment:
Roleassginments imports scalesets. Scalesets in turn imports scope

Reply:
Ref: https://github.com/kubernetes-sigs/cluster-api-provider-azure/blob/main/azure/services/scalesets/scalesets.go#L36